File: C/Lv.2/3.c
//피보나치 수
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    int a = 0, b = 1, c = 1;
    
    for(int i = 2; i <= n; i++) {
        c = (a % 1234567 + b % 1234567) % 1234567;
        a = b;
        b = c;
    }
    
    answer = c;
    
    return answer;
}

File: Python/Lv.2/3.py
//이진 변환 반복하기
def solution(s):
    answer = [];
    tmp = '';
    cnt = 0;
    cn = 0;
    
    while(1):
        cn += 1;
        tmp = '';
        for i in s:
            if (i == '1'):
                tmp += i;
            else:
                cnt += 1;
        
        st = format(len(tmp), 'b');
        s = str(st);
        
        if(len(tmp) == 1):
            break;
    
    answer.append(cn);
    answer.append(cnt);
    
    return answer